src/includes/set-release/unity.mdx

```diff
@@ -0,0 +1,18 @@
+```csharp {tabTitle:C#}
+using Sentry;
+
+// Add this to the SDK initialization callback
+options.Release = "my-project-name@2.3.12";
+```
+
+```fsharp {tabTitle:F#}
+open Sentry
+
+// Add this to the SDK initialization callback
+options.Release <- "my-project-name@2.3.12"
+```
+
+The SDK will set the release by default from `Application.productName` and `Application.version`.
```

Comment:
Same as above, we could simply add to the common content, right above the `include="set-release"`:

```
<PlatformSection supported={["unity"]}>

The SDK will set the release by default from `Application.productName` and `Application.version`.

</PlatformSection>
````

This way we tell the user: The C# snippet you'll see likely isn't relevant because the SDK takes care of this for you But if you want to override it anyway:

Reply:
I'm adding its own file so we override the Dotnet documentation that fetches the version from the assembly.